src/test/java/games/strategy/util/VersionTest.java

```diff
@@ -65,6 +65,11 @@ public void testRead3() {
   }
 
   @Test
+  public void testRead4() {
+    assertTrue("1.2.3.dev".equals(new Version("1.2.3.dev").toString()));
```

Comment:
couple nits:
- `testRead4` is not very descriptive of what is being tested. I would suggest something instead like: `verifyDevVersionToString`
- `assertTrue(a.equals(b))` can be replaced with `assertEquals(a,b)`. Two benefits, first it is more concise/direct/easier to read, second, if you get a failure then you get benefit of having both values printed.